include/libdnf/base/goal_elements.hpp

```diff
@@ -95,6 +97,23 @@ enum class GoalAction {
 enum class GoalSetting { AUTO, SET_TRUE, SET_FALSE };
 enum class GoalUsedSetting { UNUSED, USED_TRUE, USED_FALSE };
 
+struct GroupJobSettings {
+    bool ignore_case{true};
+    bool with_group_id{true};
+    bool with_group_name{true};
```

Comment:
Don't forget that dnf4 supports also translated group names.
IMO using either untranslated or translated group name as an install spec is a very bad idea.
I'd prefer to set this to false by default: `bool with_group_name{false};` or even better - disable the functionality completely. Installations should be done by `group id`.

Reply:
Right, in dnf4 the `group_id` was somehow "hidden" and `group list` command required special option `--ids` to reveal it to the user. Given that `dnf5 group list` now directly presents group ID, moving from group name  to id as primarily used identifier makes sense.